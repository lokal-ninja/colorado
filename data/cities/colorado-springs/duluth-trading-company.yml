id: 771381321
type: way
latitude: 38.90616
longitude: -104.81954
postcode: ""
city: Colorado Springs
street: "North Nevada Avenue"
housenumber: 5318
phone: ""
opening_hours: ""
website: ""