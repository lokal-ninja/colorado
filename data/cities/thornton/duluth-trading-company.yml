id: 6990667267
type: node
latitude: 39.99743
longitude: -104.98033
postcode: "80023"
city: Thornton
street: "Washington Street"
housenumber: 16557
phone: ""
opening_hours: ""
website: ""