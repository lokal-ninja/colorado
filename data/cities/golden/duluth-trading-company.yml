id: 183653353
type: way
latitude: 39.7245
longitude: -105.18594
postcode: "80401"
city: Golden
street: "West Colfax Avenue"
housenumber: 16630
phone: "+17206367660"
opening_hours: ""
website: "https://www.duluthtrading.com/locations/?StoreID=042&street-address=16630WColfaxAve.&city=Golden&srccode=WBGMB19"